api/src/main/resources/messages_fr.properties

```diff
@@ -71,3 +71,24 @@ ${project.parent.artifactId}.report.newEpisodesOfDiseases.females.label=F
 
 ${project.parent.artifactId}.report.lab.reportName=MSPP: Examens de laboratoire
 ${project.parent.artifactId}.report.lab.reportDescription=Rapport des Examens de laboratoire demandé par le MSPP à Haiti
+${project.parent.artifactId}.report.antenatal.reportName=Prise en charge de la mere
+${project.parent.artifactId}.report.antenatal.reportDescription=MSPP (Antenatal) report required by Haitian MoH to any clinic in Haiti
```

Comment:
`Rapport MSPP (prénatal) requis par le ministère de la Santé haïtien à toute clinique en Haïti`

Reply:
Done

Reply:
Actually the best sentence I came up to was:
"Rapport Anténatal demandé par le MSPP à Haiti" (that's how it's set for all other merged reports)